specification/resources/resource-manager/Microsoft.Resources/preview/2022-11-01-preview/snapshots.json

```diff
@@ -0,0 +1,247 @@
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{resourceProviderNamespace}/{resourceType}/{resourceName}/providers/Microsoft.Resources/snapshots/{snapshotId}": {
+      "get": {
+        "tags": [
+          "GetSnapshot"
+        ],
+        "operationId": "Snapshots_Get",
+        "x-ms-examples": {
+          "GetSnapshot": {
+            "$ref": "./examples/GetSnapshot.json"
+          }
+        },
+        "description": "Obtains the specified snapshot for the target resource",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceProviderNamespaceParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceTypeParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/snapshotIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "$ref": "#/definitions/Snapshot"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Snapshot": {
+      "type": "object",
+      "description": "A generic resource snapshot with additional metadata",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/SnapshotProperties"
+        }
+      }
+    },
+    "SnapshotProperties": {
+      "description": "The properties of a snapshot",
+      "type": "object",
+      "properties": {
+        "apiVersion": {
+          "type": "string",
+          "description": "The API Version of the resource captured"
+        },
+        "content": {
+          "description": "The actual ARM Resource",
+          "type": "object"
+        }
+      }
+    },
+    "SnapshotListResult": {
+      "description": "The list of snapshots",
+      "type": "object",
+      "properties": {
+        "nextLink": {
+          "type": "string",
+          "description": "The link used to get the next page of Snapshots"
+        },
+        "value": {
```

Comment:
>value [](http://example.com/codeflow?start=9&length=5)

This looks like a good place to use x-ms-identifier, to specify that the snapshots in the list are distinguishable by their snapshot id